gallery/src/Application.tsx

```diff
@@ -4,92 +4,116 @@
-              </Route>
-              <Redirect to="/" />
-            </Switch>
-          </Route>
-        </Switch>
+  return (
+    <div id="temporary-navigation-sidebar-example" className="nx-page-content nx-page-content--full-width">
+      <NxStatefulNavigationSidebar isDefaultOpen={true}
+                                   toggleOpenIcon={faArrowLeft}
+                                   toggleCloseIcon={faCrow}
+                                   logoAltText="product name"
+                                   logoLink="/home"
+                                   logoImg={logo}>
+        <NxNavigationSidebarLinks>
+          <a href="#alpha" className="nx-sidebar-navigation__link selected nx-text-link">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#beta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#gamma" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#delta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#epsilon" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#Zeta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#Etha" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#Theta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <NxOverflowTooltip>
```

Comment:
Is this needed or would the expectation be that designers just don't design any link names that are long enough to overflow?  I'm assuming the expanded sidebar width will always be constant in any given app.

Reply:
> Is this needed or would the expectation be that designers just don't design any link names that are long enough to overflow? I'm assuming the expanded sidebar width will always be constant in any given app.

This was a requirement in Stepheni's old mocks — I've included it as part of my new Link component in 3854748e07cbcb5be6eb4098f549da2271b0fde5